cvcl/templates/assignment_create.html

```diff
@@ -4,6 +4,8 @@
 {% block content %}
     <h1>Create Assignment</h1>
 
+    <br>
```

Comment:
That's not spacing, that's new line. You should add a margin on h1 instead, in CSS.